src/Avalonia.Visuals/Vector.cs

```diff
@@ -333,5 +333,24 @@ public static Vector UnitX
         /// </summary>
         public static Vector UnitY
             => new Vector(0, 1);
+
+        /// <summary>
+        /// Deconstructor for decomposition Vector
+        /// </summary>
+        /// <param name="x">The X component.</param>
+        /// <param name="y">The Y poscomponentition.</param>
```

Comment:
Typo

Reply:
Thank you @VisualMelon 

I update this PR according to your comments and suggestions